.github/workflows/experimental_tests.yml

```diff
@@ -30,6 +30,12 @@ jobs:
       # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
 
+      - name: Set up MinGW
```

Comment:
Was the test failing without mingw installled?

Reply:
yeah, it found a GCC binary, but that GCC binary was failing, even with just pure scons build. No error messages.